### PR TITLE
[FR] difficulty

### PR DESCRIPTION
**Pull request by Gastonia02**
main_fr-1-37-38_page-0001
main_fr-1-37-38_page-0002

### CODE REVIEW

**Comment by DarthGandalf** [comment]
Hi, FYI in #272 I'm replacing the tables with ones which can be easily translated without trying to edit images

File: translations/difficulty.tex/fr.po [hunk 1]
```diff
@@ -50,10 +51,9 @@ msgstr ""
 #.  !TeX spellcheck = en_US
 #. type: Plain text
 #: sections/difficulty.tex:3
-#, fuzzy
 #| msgid "\\addsection{AI Heroes}{\\spells/fortune.png}"
```
Comment (qwrtln) [inline translations/difficulty.tex/fr.po:54]:
```suggestion
```

File: translations/difficulty.tex/fr.po [hunk 2]
```diff
@@ -122,6 +136,36 @@ msgid ""
 "would normally go to your hand now go immediately to your discard pile "
 "instead.} \\\\ \\end{tabularx} }"
 msgstr ""
+"\\hommtable{38}{ \\centering \\textbf{Table de variantes} \\small "
+"{ \\color{white} Vous pouvez modifier ces règles pour augmenter ou diminiuer "
```
Comment (qwrtln) [inline translations/difficulty.tex/fr.po:140]:
```suggestion
"\\hommtable{38}{ \\centering \\textbf{Table de variantes}\\\\ \\small "
"{ \\color{white} Vous pouvez modifier ces règles pour augmenter ou diminiuer\\\\ "
```
This should make at least the header right.

**Comment by qwrtln** [comment]
The table still looks off. I'm making a PR to make it look better, perhaps it will help.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**